public/stylesheets/site/2.0/26-media-narrow.css

```diff
@@ -90,10 +90,6 @@ body .narrow-shown {
   background-position: center;
 }
 
-#main.session {
```

Comment:
It looks like some recent changes got clobbered in a merge conflict -- the stuff that's being removed here should stay.